src/containers/WebComponentLoader.jsx

```diff
@@ -46,7 +46,10 @@ const WebComponentLoader = (props) => {
   const { t } = useTranslation();
   const [projectIdentifier, setProjectIdentifier] = useState(identifier);
   localStorage.setItem("authKey", authKey);
-  const user = useSelector((state) => state.auth.user);
+  const localStorageUser = authKey
```

Comment:
(regarding your comment about this) I think this is fine since authKey is passed in as an argument to the component, and can only be updated on render

Reply:
Great! 😎